docker/package/test-ubuntu-binaries.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-FileCopyrightText: 2021 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import subprocess
+import sys
+
+binaries = [
+    "tezos-client",
+    "tezos-admin-client",
+    "tezos-node",
+    "tezos-signer",
+    "tezos-codec",
+    "tezos-baker-PtMumbai",
+    "tezos-accuser-PtMumbai",
+    "tezos-smart-rollup-client-PtMumbai",
+    "tezos-smart-rollup-node-PtMumbai",
+    "tezos-baker-PtNairob",
+    "tezos-accuser-PtNairob",
+    "tezos-smart-rollup-client-PtNairob",
+    "tezos-smart-rollup-node-PtNairob",
+]
+
+
+def setup():
+    commands = [
+        "apt update -y",
+        "apt install -y software-properties-common",
+        "apt update -y",
+        "add-apt-repository -y ppa:serokell/tezos",
+        "apt-get update -y",
```

Comment:
I believe all this commands (except one `apt update -y`) should be moved to the build stage in the docker file

Reply:
Currently, you can run:
* tests on your local machine
* tests on your local machine inside a docker container
* tests on CI inside a docker container

If we remove these updates from the Python script, we want to be able to use the first variant. Not sure, if we need that variant, but still.